src/MyHealth/SymptomLogContext.tsx

```diff
@@ -8,50 +8,20 @@ import React, {
 
 import {
   SymptomLogEntry,
-  DailyCheckIn,
+  CheckIn,
   CheckInStatus,
   DayLogData,
-  serializeDailyLogData,
+  combineSymptomAndCheckInLogs,
 } from "./symptoms"
-
-const fetchLogEntries = (): Promise<SymptomLogEntry[]> => {
-  return Promise.resolve([
-    {
-      id: "1",
-      symptoms: ["Loss of Breath"],
-      date: 1600614626042,
-    },
-    {
-      id: "2",
-      symptoms: ["Cough", "Fever"],
-      date: 1600610400000,
-    },
-    {
-      id: "3",
-      symptoms: ["Fever"],
-      date: 1599919200000,
-    },
-  ])
-}
-
-const fetchDailyCheckIns = (): Promise<DailyCheckIn[]> => {
-  return Promise.resolve([
-    {
-      date: 1600804626042,
-      status: CheckInStatus.FeelingGood,
-    },
-    {
-      date: 1599919200000,
-      status: CheckInStatus.FeelingNotWell,
-    },
-  ])
-}
+import { getLogEntries, getCheckIns, addCheckIn } from "../gaen/nativeModule"
```

Comment:
```suggestion
import { getLogEntries, getCheckIns, addCheckIn } from "./nativeModule"
```

what do you think about creating a new NativeModule scoped to the MyHealth feature? this doesnt seem directly related to GAEN.

Reply:
We are moving the modules in a future iteration. 👍